src/docs.js

```diff
@@ -6,38 +6,48 @@ let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
 
 async function documenter(options) {
-
   options = _.defaults({}, options, {
+    metadata: null,
     schemas: [],
     docsFolder: null,
+    references: [],
   });
   assert(options.schemas, 'options.schemas must be given');
   assert(options.schemas instanceof Array, 'options.schemas must be an array');
 
-  let schemas = options.schemas;
   let tarball = tar.pack();
+  let metadata = options.metadata;
+
+  if (metadata) {
+    let data = JSON.stringify(metadata, null, 2);
+    tarball.entry({name: 'metadata.json'}, data);
+  }
 
-  // add schemas to tarball
+  let schemas = options.schemas;
   schemas.forEach(schema => {
     let data = JSON.stringify(schema, null, 2);
-    tarball.entry({name: 'schema/' + schema.id}, data);
+    let schemaFilename = Object.keys(schema)[0];
```

Comment:
I did make that change. It's possible that the version in this library hasn't been updated yet?